src/pages/faq/cors.md

```diff
@@ -0,0 +1,250 @@
+---
+previousText: 'Native Errors'
+previousUrl: '/docs/faq/native'
+nextText: 'Developer Tips'
+nextUrl: '/docs/faq/tips'
+---
+
+# CORS Errors
+
+
+## What is CORS?
+
+**Cross-Origin Resource Sharing (CORS)** is a mechanism that browsers and webviews — like the ones powering Capacitor and Cordova — use to restrict HTTP and HTTPS requests made from scripts to resources in a different origin for security reasons, mainly to protect your user's data and prevent attacks that would compromise your app.
+
+In order to know if an external origin supports CORS, the server has to send some <a href="#cors-headers">special headers</a> for the browser to allow the requests.
+
+An **origin** is the combination of the **protocol**, **domain**, and **port** from which your Ionic app or the external resource is served. For example, apps running in Capacitor have `capacitor://localhost` (iOS) or `http://localhost` (Android) as their origin, and Cordova apps may run from `http://localhost:8080` or the `file://` protocol.
```

Comment:
On WebView 3.x it uses `http://localhost` without port for Android, and `ionic://localhost` for iOS

Reply:
Moved it to the solutions section, thanks!